EDDI/EDDI.cs

```diff
@@ -1616,14 +1624,19 @@ private bool eventMarket(MarketEvent theEvent)
 
         private bool eventOutfitting(OutfittingEvent theEvent)
         {
-            if (allowOutfittingUpdate)
+            // Don't proceed if we've already viewed outfitting while docked or when loading pre-existing logs
+            if (allowOutfittingUpdate && !theEvent.fromLoad)
             {
                 OutfittingInfoReader info = OutfittingInfoReader.FromFile();
                 if (info.Items != null && info.MarketID == theEvent.marketId
                     && info.StarSystem == theEvent.system
                     && info.StationName == theEvent.station
                     && info.Horizons == Instance.inHorizons)
                 {
+                    // Post an update event for new outfitting data
+                    enqueueEvent(new MarketInformationUpdatedEvent(info.timestamp, theEvent.system, theEvent.station, theEvent.marketId, null, null, info.Items.Select(i => i.name).ToList(), null, inHorizons));
+
+                    // Update the current station outfitting
                     List<EddiDataDefinitions.Module> modules = new List<EddiDataDefinitions.Module>();
                     foreach (OutfittingInfo item in info.Items)
```

Comment:
This block could be simplified with LINQ.
```
var modules = info.Items
    .Select( i => EddiDataDefinitions.Module.FromOutfittingInfo(i))
    .Where( i => i ! null )
    .ToList();
```

Reply:
ok